tests/integration/pilot/common/routing.go

```diff
@@ -657,6 +658,36 @@ spec:
 		workloadAgnostic: true,
 	})
 
+	t.RunTraffic(TrafficTestCase{
+		name: "fault abort gRPC",
```

Comment:
we should try setting sourcematchers and targetMatchers to `match.NotNaked, match.NotHeadless`.

Currently we have proxyless grpc opted out; this seems like a REALLY useful feature for proxyless grpc.

I don't know if itworks though, so not certain that will pass

Reply:
Did it. 
```go
		sourceMatchers:   []match.Matcher{match.NotNaked, match.NotHeadless},
		targetMatchers:   []match.Matcher{match.NotNaked, match.NotHeadless},
```
And it is 🟢
![image](https://user-images.githubusercontent.com/41095737/172416657-0e05b211-81d0-48ec-9c1b-d4bfb5aa70b1.png)

Let's enable it?